core/proto/keytransparency_v1_proto/keytransparency_v1_admin_proto.proto

```diff
@@ -20,30 +20,74 @@ package keytransparency.v1.proto;
 
 import "trillian.proto";
 import "crypto/keyspb/keyspb.proto";
-import "proto/keytransparency_v1_proto/keytransparency_v1_proto.proto";
 import "google/protobuf/duration.proto";
+import "google/rpc/status.proto";
 
-// BatchUpdateEntriesRequest will update the accounts of the given user_ids to be set to
-// user data if those accounts have key_id in their set of authorized_keys.
-message BatchUpdateEntriesRequest {
-  // users is a map from user_ids to user data.
-  map<string,UserProfile> users = 1;
+// Account represents plain account information that gets committed to and obfuscated in Entry.
+message Account {
+  // domain_id specifies the domain.
+  string domain_id = 1;
+  // app_id specifies that application.
+  string app_id = 2; 
+  // user_id specifies the user.
+  string user_id = 3;
+  // data is the public key material for this account.
+  bytes data = 4;
+  // authorized_keys is the set of keys allowed to sign updates for this entry.
+  repeated keyspb.PublicKey authorized_keys = 5;
+}
 
-  // app_id is the application to make this change for.
-  string app_id = 2;
+// GetAuthorizedKeysRequest is empty. 
+// TODO(gbelvin): Should we shard these keys by domain_id and app_id? 
+message GetAuthorizedKeysRequest{}
+
+// GetAuthorizedKeysResponse contains the set of public keys that correspond 
+// to the private keys the AccountRecoveryService has access to.
+message GetAuthorizedKeysResponse{
+  // AccountRecoveryKeys is a map from pub_key_id to the public key itself.
+  map<string, keyspb.PublicKey> account_recovery_keys = 1;
```

Comment:
I can't see some of this information just from the Account entity (pub_key_id, active key). It should probably be available there too. An option would be an entity that represents pub_key_id + keyspb.PublicKey, so you'd have a `repeated` field both here and at Account.